### ISSUE TITLE
SVG test paths-data-20-f-manual.svg should check negative rx/ry

### ISSUE THREAD

**Issue by tshinnic** [w3c/web-platform-tests#6195]
In all SVG versions, the [elliptical arc implementation notes](https://www.w3.org/TR/SVG2/implnote.html#ArcOutOfRangeParameters) mention that 
> If rx or ry have negative signs, these are dropped; the absolute value is used instead.

Test `paths-data-20-f-manual.svg` checks multiple assertions about the elliptical arc command, such as syntax and value restrictions, e.g. flag values.

However it does not test the above implementation note regarding rx/ry values.  Every test of the elliptical arc in the test uses the same legal values, "a25,25 0 ....".  I have tested example 'bad' values such as "a-25,-25 0 ...." and this works in several major browsers, indicating that they do conform to the implementation note.  

Note also that the [grammar](https://www.w3.org/TR/SVG2/paths.html#PathDataBNF) for elliptical arc has now been changed in SVG 2.0 to omit insisting on `nonnegative_number` for rx/ry as confusingly seen in SVG 1.1 and 1.0 .

Test `paths-data-20-f-manual.svg` **should have additional checks** added to verify conformance to the implementation notes with respect to negative rx/ry values on elliptical arc commands.

Note also that no checking in this test is done with respect to variant rotation values.

Note also that the grammar has changed in 2.0 to add rule `elliptical_arc_closing_argument` that omits the final coordinate pair.  There is no test for this, but then there is no prose supporting this change either.  Oh my....

**Comment by AmeliaBR** [comment]
All good suggestions. 

> Note also that the grammar has changed in 2.0 to add rule `elliptical_arc_closing_argument` that omits the final coordinate pair. There is no test for this, but then there is no prose supporting this change either. Oh my....

The syntax with the final coordinate pair missing is only valid before a `z`/`Z` command [and is described in the prose for that command](https://www.w3.org/TR/SVG2/paths.html#PathDataClosePathCommand).

**Comment by tshinnic** [comment]
Yes, I found that prose after a bit more searching, but which prompted another issue https://github.com/w3c/svgwg/issues/325 against the grammar implementing "closepath in lieu", and which looks wrong for a couple cases.

All progress is uneven...  (And I have been mentioning your missive from April to several people)  Wishing you *eventual* blessings for your efforts!

**Comment by AmeliaBR** [comment]
I should have mentioned that there were other errors discovered in the path grammar after SVG 2 candidate recommendation was published.  The most up-to-date version [is in the editor's draft](https://svgwg.org/svg2-draft/paths.html#PathDataBNF).
